rfcs/0060_UMA_Unified_Modular_Accelerator_Interface.md

```diff
@@ -388,3 +388,100 @@ Below is a list of currently planned configuration parameters for the initial ve
 |Parameter|Type|Description|
 |---------|----|-----------|
 |merge_compiler_regions|bool (default: True)|Enables/disables the `MergeCompilerRegions` pass during partitioning.|
+
+
+## TVMC integration
+
+Proposals:
+  - Add uma_cli functionality that is currently only used for the tutorial
+  - possibility to compile with tvmc
```

Comment:
One detail that will make the API to be more CLI-auto-generated friendly for UMA targets would actually be offering a description field while adding new target.

```
# Target configuration
        self._register_target_attr("dimension", description="Some description string.", default=8)
```